prow/oss/cluster/prow-controller-manager.yaml

```diff
@@ -22,32 +21,32 @@ spec:
       - name: prow-controller-manager
         image: gcr.io/k8s-prow/prow-controller-manager:v20201026-f7d653694b
         args:
-        - --config-path=/etc/config/config.yaml
+        - --config-path=/etc/config/config-20201002
```

Comment:
I think the line needs `20201002` is `- --kubeconfig=/etc/kubeconfig/config-20201002`, which is line # 28

Reply:
Done

Reply:
Ugh, no, one sec

Reply:
Done